tests/core/management/commands/test_reset_deepl_budget.py

```diff
@@ -0,0 +1,83 @@
+from datetime import datetime
+
+import pytest
+from django.core.management.base import CommandError
+
+from integreat_cms.cms.models import Region
+
+from ..utils import get_command_output
+
+
+def test_not_first_day():
+    """
+    Ensure that the command will not run when it's not the 1st day of the month without --force
+    """
+    with pytest.raises(CommandError) as exc_info:
+        assert not any(get_command_output("reset_deepl_budget"))
+        if datetime.now().day != 1:
+            assert (
+                str(exc_info.value)
+                == "It is not the 1st day of the month. If you want to reset DeepL budget despite that, run the command with --force"
+            )
```

Comment:
```suggestion
from unittest.mock import patch

import pytest
from django.core.management.base import CommandError

from integreat_cms.cms.models import Region

from ..utils import get_command_output


class datetime_not_first_day:
    """
    Fake datetime object where now() is never the first day of the month
    """

    @classmethod
    def now(cls):
        real_now = datetime.now()
        return real_now.replace(day=2)


@patch("datetime.datetime", datetime_not_first_day)
def test_not_first_day():
    """
    Ensure that the command will not run when it's not the 1st day of the month without --force
    """
    with pytest.raises(CommandError) as exc_info:
        assert not any(get_command_output("reset_deepl_budget"))
        assert (
            str(exc_info.value)
            == "It is not the 1st day of the month. If you want to reset DeepL budget despite that, run the command with --force"
        )
```
This test case will fail on the first day of the month since it will not raise the given exception.
We could solve this by a conditional, but I would prefer if we could patch the datetime class to just never return the first day of the month...